docs/DEVSETUP.md

```diff
@@ -1,70 +1,153 @@
+This CRA allows developers to view the application in a web browser, make changes to the code, and see those changes reflected immediately without having to manually refresh the page. This feature is known as "hot reloading".
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+You can learn more in the [Create React App documentation](https://create-react-app.dev/docs/getting-started/).
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+To learn more about React, check out the [React documentation](https://react.dev/).
 
-### `npm test`
+## Setting Up
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+It is advisable to work on this project using an IDE rather than through the github repository workspace; as this makes it easier to make changes to this project using git commands.
 
-### `npm run build`
+Follow these next steps to setup your local repository in your IDE.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Clone this repository <a name="clone"></a>
+This simply means replicating the files in this project in your IDE workspace.
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### Step 1
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Navigate to your forked repository, on the left side of your screen click on the drop down with "code" written on it. This will pop up a small dialog box; make no changes to this box but simply copy the url.
 
-### `npm run eject`
+![](./Clone.png)
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+### Step 2
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+Create a folder on your computer and open this folder in your IDE. Open the terminal in your IDE and cd into this folder
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+Run the command
 
-## Learn More
+```
+git init
```

Comment:
Hii @gbowne1 @Chizobaonorh 
Is it necessary to run``` git init```?

Reply:
no, it is not.  This part of the documentation can be removed.  In fact, if one tried to do that in the project, it would be detrimental to the project.

Reply:
I always initialize my git, but i can remove it since its not needed for this project.

Reply:
I see. Running "git init" is for setting up a "fresh" repo locally. Not when you are cloning or "copying" an existing repo locally.

Reply:
i have made the final changes.

Reply:
Yes git init is only for when the project gets initially set up not after cloning.